package.json

```diff
@@ -144,22 +138,59 @@
     "parser": "babel-eslint",
     "env": {
       "es6": true,
-      "node": true
+      "node": true,
+      "jest": true
     }
   },
   "eslintIgnore": [
-    "**/node_modules/**",
-    "**/template-*/**",
-    "**/__tests__/fixtures/**",
-    "**/spec/**"
+    "**/.tmp/**"
   ],
   "workspaces": [
     "packages/*"
   ],
   "jest": {
+    "collectCoverage": true,
+    "coverageProvider": "v8",
+    "testPathIgnorePatterns": [
+      "/node_modules/",
+      "<rootDir>/packages/spec/.tmp/",
```

Comment:
We could as well ignore all the `.tmp`-folders, like the eslint config is doing it.